src/frontend/utils/dato.ts

```diff
@@ -88,3 +89,9 @@ export const erGyldigDato = (dato: string | Date): boolean =>
 
 export const plusDager = (dato: string | Date, antallDager: number): string =>
     tilLocaleDateString(addDays(tilDato(dato), antallDager));
+
+export const formaterÅrMåned = (dato: string): string => {
+    const str = format(parseISO(dato), 'MMM yyyy', { locale: nb });
+
+    return str.charAt(0).toUpperCase() + str.slice(1);
```

Comment:
Lurer nesten på om `str.charAt(0).toUpperCase()` burde være et annet sted, eller en egen metode med et navn som `formaterÅrMånedMedStorBokstav` eller noe slik
For å unngå at noen bruker den et sted der det ikke skal være stor bokstad 😁